internal/authflow/flow.go

```diff
@@ -69,7 +75,7 @@ func authFlow(oauthHost string, IO *iostreams.IOStreams, notice string, addition
 	debugEnabled, debugValue := utils.IsDebugEnabled()
 	if debugEnabled {
 		logTraffic := strings.Contains(debugValue, "api")
-		httpClient.Transport = api.VerboseLog(IO.ErrOut, logTraffic, IO.ColorEnabled())(httpClient.Transport)
+		httpClient.Transport = verboseLog(IO.ErrOut, logTraffic, IO.ColorEnabled())(httpClient.Transport)
```

Comment:
For `oauth.Flow` purposes, a httpClient is anything that responds to `PostForm(string, url.Values) (*http.Response, error)`. Instead of using `http.DefaultClient`, do you think we could wrap a go-gh client (with verbose logging) and wrap it so that it responds to an additional method?

Reply:
I like this idea, but I think it might be a bit out of scope for right now as it would involve making changes to `cli/oauth` repo.